community-generators/typegraphql-prisma/generated/typegraphql-prisma/enhance.js

```diff
@@ -0,0 +1,104 @@
+"use strict";
```

Comment:
intentional to commit these files?

Reply:
Yes, their example does that as well. We can remove it in this context as they can mislead. Separate issue/problem though.

Reply:
https://github.com/prisma/e2e-tests/issues/1319 will pick it after the package change has settled.